scenario/CometScenario.ts

```diff
@@ -1,7 +1,7 @@
 import { CometProperties, scenario } from './context/CometContext';
 import { expect } from 'chai';
 
-scenario('initializes governor correctly', { migrate: true }, async ({ comet, timelock, actors }, world) => {
```

Comment:
Does this currently fail with MigrationConstraint? We could also just skip the scenario for now

Reply:
I don't think so? But I changed `MigrationConstraint` to always add solutions for any 'modified' migration scripts

Reply:
So in other words no, this just isn't a requirement for `MigrationConstraint` anymore